data/json/recipes/recipe_deconstruction.json

```diff
@@ -497,6 +497,7 @@
     "reversible": true,
     "qualities": [ { "id": "SCREW","level": 1 } ],
     "components": [
+      [ [ "small_lcd_screen", 1 ] ],
```

Comment:
Well, you can get one by disassembling a PDA.

Though I also realized that LCD screens don't work well when cut up, so 1 is fine.